js/index.js

```diff
@@ -1 +1,51 @@
 // Your code goes here
+
+// Query Selectors
+const bodyQuery = document.querySelector('body');
+const navLinks = document.querySelectorAll('nav a');
+const imgSelector = document.querySelectorAll('img');
+lowerButtons = document.querySelectorAll('.btn');
+const navLink = document.querySelector('header nav a');
+
+// Copy Alert [X]
+bodyQuery.addEventListener('copy', event => {
+    event.preventDefault();
+    alert('Plagurism is a crime!')
+});
+
+// Scroll event [X]
+window.addEventListener('scroll', event => {
+    event.preventDefault();
+    let body = document.body;
+    body.classList.toggle('cursorClass');
+});
+
+// Nav mouseover [X]
+navLinks.forEach(a => {
+    a.addEventListener('mouseover', event => {
+        event.target.style.textDecoration = 'underline';
+    })
+});
+
+// Blur mouseover on photos [X]
+imgSelector.forEach(item => 
+    item.addEventListener('mouseover', event => {
+        event.preventDefault();
+        event.target.style.filter = 'blur(8px)';
+    })
```

Comment:
`preventDefault` is not necessary in all cases. You usually only use it on forms and links that you don't want to perform their default action. Good consistency, though! 👍